### PR TITLE
Fix project build.

### PR DESCRIPTION
**Pull request by karelmaxa**
This PR fixes build of the project. The following significant changes have been performed:

* Switch to the latest releases of the `wrensec-parent` and `wrensec-commons`.
* Fixed the UI build (including switching to the latest release of the `wrensec-ui`).
* Cleaned up all POM files.
* Fixed tests after Mockito upgrade.
* Introduced Docker build.
* Several unit tests have been temporarily disabled (there is a problem with testing LDAP server).

### CODE REVIEW

**Review by pavelhoral — CHANGES_REQUESTED** [review]
Just a few minor nitpicks.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+FROM tomcat:9-jdk17-temurin
+
+# Set environment variables
+ENV \
+  WRENIG_HOME="/srv/wrenig" \
+  JAVA_OPTS=" \
+    -Dopenig.base=/srv/wrenig \
+  " \
+  CATALINA_OPTS="-server -Xmx1g"
```
[the remark this answers is not in the document]
Comment (karelmaxa) [reply Dockerfile]: Changed.

File: openig-core/pom.xml [hunk 2]
```diff
@@ -1,199 +1,248 @@
...
+
+        <dependency>
+            <groupId>io.dropwizard.metrics</groupId>
+            <artifactId>metrics-core</artifactId>
+            <version>${dropwizard-metrics.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.apache.ivy</groupId>
+            <artifactId>ivy</artifactId>
```
[the remark this answers is not in the document]
Comment (karelmaxa) [reply openig-core/pom.xml]: Actually, I don't know. There is probably no use case at the moment. I have removed the Ivy dependency and everything seems to work fine.

**Review by pavelhoral — APPROVED** [review]
LGTM

**Pull request merged**